client/utils/use-payment-method-currencies.js

```diff
@@ -154,16 +154,63 @@ const getWechatPayCurrencies = () => {
 	return upeCurrencies;
 };
 
+// Returns the specific currencies Klarna supports for the corresponding Stripe account based on location.
+// Documentation: https://docs.stripe.com/payments/klarna#:~:text=Merchant%20country%20availability.
+const getKlarnaCurrencies = () => {
```

Comment:
Minor: maybe we could replace the "US" case with the default clause. Or throw and error if any matches the options just to be safe.

Reply:
👍 I've decided to throw an error rather than default to USD. I think returning USD may suppress the error and make error troubleshooting more difficult. [de21720](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3235/commits/de21720f8f51ad0007c60204e27416dad206994a)